operatorcourier/cli.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 import sys
 from operatorcourier import api
+import re
```

Comment:
can we order imports like:

1. stdlib
2. third party
3. package dependencies